api/v1alpha/api.proto

```diff
@@ -201,6 +204,9 @@ message PodFilter {
 
         // If not empty, the pods that have all of the annotations will be returned.
         repeated KeyValue annotations = 6;
+
+        // If not empty, the pods whose cgroup are listed will be returned.
```

Comment:
look up by cgroup?  Cadvisor as it turns out, especially with systemd run rkt pods, that they can be wherever they want to be (i.e. can have their own slice defined....)